ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiLedgerConfigurationService.scala

```diff
@@ -13,33 +13,38 @@ import com.daml.grpc.adapter.ExecutionSequencerFactory
 import com.daml.ledger.api.domain.LedgerId
 import com.daml.ledger.api.v1.ledger_configuration_service._
 import com.daml.logging.{ContextualizedLogger, LoggingContext}
+import com.daml.logging.LoggingContext.withEnrichedLoggingContext
 import com.daml.platform.api.grpc.GrpcApiService
 import com.daml.platform.server.api.validation.LedgerConfigurationServiceValidation
 import io.grpc.{BindableService, ServerServiceDefinition}
 
 import scala.concurrent.ExecutionContext
 
 private[apiserver] final class ApiLedgerConfigurationService private (
-    configurationService: IndexConfigurationService)(
+    configurationService: IndexConfigurationService,
+)(
     implicit protected val esf: ExecutionSequencerFactory,
     protected val mat: Materializer,
-    logCtx: LoggingContext)
-    extends LedgerConfigurationServiceAkkaGrpc
+    loggingContext: LoggingContext,
+) extends LedgerConfigurationServiceAkkaGrpc
     with GrpcApiService {
 
   private val logger = ContextualizedLogger.get(this.getClass)
 
   override protected def getLedgerConfigurationSource(
-      request: GetLedgerConfigurationRequest): Source[GetLedgerConfigurationResponse, NotUsed] =
-    configurationService
-      .getLedgerConfiguration()
-      .map(
-        configuration =>
-          GetLedgerConfigurationResponse(
-            Some(LedgerConfiguration(
-              Some(toProto(configuration.maxDeduplicationTime)),
-            ))))
-      .via(logger.logErrorsOnStream)
+      request: GetLedgerConfigurationRequest,
+  ): Source[GetLedgerConfigurationResponse, NotUsed] =
+    withEnrichedLoggingContext(Map.empty[String, String]) { implicit loggingContext =>
```

Comment:
I would have assumed calling `withEnrichedLoggingContext` with an empty map would be a no-op, but you seem to be doing it a lot. Why?

Reply:
WIP left-over, I'll address this as part of the [comments I left](https://github.com/digital-asset/daml/pull/6924#discussion_r463516895) in this regard.